envi/tests/test_arch_arm.py

```diff
@@ -5,58 +5,594 @@
 import envi.registers as e_reg
 import envi.memcanvas as e_memcanvas
 import envi.memcanvas.renderers as e_rend
-import envi.archs.h8 as e_h8
+import envi.archs.arm as arm
 import vivisect
 import platform
-import unittest
+import unittest  
 from envi import IF_RET, IF_NOFALL, IF_BRANCH, IF_CALL, IF_COND
-from envi.archs.h8.regs import *
-from envi.archs.h8.const import *
-from envi.archs.h8.parsers import *
+from envi.archs.arm.regs import *
+from envi.archs.arm.const import *
+from envi.archs.arm.disasm import *
+import binascii   # temporarily included for testing purposes only - allows to print out binary values
 
 
-# OPHEX, VA, repr, flags, emutests
-instrs = [
```

Comment:
Not sure I fully understand what you say is missing.

 If you are referring to the list missing items it SHOULD be the full IDA dump converted to be compatible with this format. If there are items missing from the list then they are missing from the original IDA dump. Luckily I saved the helper module in case we needed to do any further IDA dumps. 

I also saved the original dump listing. 
There are no straight LDR lisings in the IDA dump I have. Just modifications of it such as LDRH



Date: Fri, 3 Jun 2016 06:57:37 -0700
From: notifications@github.com
To: vivisect@noreply.github.com
CC: redheadedrod@hotmail.com; author@noreply.github.com
Subject: Re: [atlas0fd00m/vivisect] Update unit tests for opcodes (#3)

In envi/tests/test_arch_arm.py:

>  
>  
> -# OPHEX, VA, repr, flags, emutests
> -instrs = [

these?  ldr r3 doesn't show up in your list at all.


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.